site/src/scaffold/scripts/popover.ts

```diff
@@ -0,0 +1,52 @@
+export function initPopovers(container: HTMLElement) {
+	container.addEventListener('click', (event: Event) => {
```

Comment:
thinking this could be toggle since we only care when the details menu is opening at this point in the code.

Reply:
I finally figured out why it didn't work for me on my previous attempts, updated!